### ISSUE TITLE
Page Jumps jumping too far down

### ISSUE THREAD

**Issue by carladoria** [Automattic/wp-calypso#46627]
When clicking a link that is linked to an anchor tag (Page jumps), the jump takes you to below the Heading/text where the anchor tag is located. 
#### Steps to reproduce

![ezgif-3-da7b90eb5d31](https://user-images.githubusercontent.com/23703267/96654011-4e582e00-1308-11eb-9d89-73e218bda1a4.gif)

What I expected:

I expected the Page Jump to take you to the respective section and still show the heading, i.e. the page jump shouldn't go further down the text that has the Anchor tag. 

This was also brought up in 721-gh-en.support-docs-content  that made reference to a[ CSS request ](p1594910897307800-slack-C025Q5RK2), but it seems we should consider improving the Page jump in anchor tags in general.  
-->

**Comment by carladoria** [comment]
a report in 26175662-hc

**Comment by obenland** [comment]
@carladoria I'm having a hard time identifying in which environment you discovered this bug. Is this on the front-end of a site?

**Comment by carladoria** [comment]
@obenland Yes, it shows on the front end. The page jump works, that is why I reported this as an enhancement. We always have users not happy with the page jump "jumping" to below the heading/text that has the anchor tag, when the expected is to see in the screen the heading/text at the top. 
Let me know if you need more details. 
If it helps, I can always see this in Chrome.

**Comment by obenland** [comment]
Ah, that is helpful. Do you happen to remember which theme it was?
